Renamed FRS -> Azure Fluid Relay (#7343)
microsoft/FluidFramework: examples/hosts/app-integration/external-controller/README.md

- SharedDirectory - root

## Backed Locally and running with live Azure Fluid Relay service instance

> We can connect to both a live Azure Fluid Relay  instance by passing in the tenant ID, orderer and storage, as well as using the tenant ID as "local" for running against Tinylicious for development purpose.

Reviewer: ```suggestion
We can connect to a live Azure Fluid Relay instance by passing in the tenant ID, orderer, and storage, or we can connect to a local Tinylicious server for development purposes by passing in "local" for the tenant ID.
```

```

In this way, we can toggle between remote and local mode using the same config format. We make use of

> `AzureFunctionTokenProvider` for running against live Azure Fluid Relay  instance since it is more secured, without exposing the tenant

Reviewer: ```suggestion
`AzureFunctionTokenProvider` for running against live Azure Fluid Relay instance since it is more secured, without exposing the tenant
```
